.gitignore

```diff
@@ -1,2 +1,4 @@
+dist/
+esm/
```

Comment:
I'm not sure about removing `esm/` either. An app requiring the package will not require them? Check the definition in `package.json`:

```json
"main": "./dist/index.js",
  "module": "./esm/index.js",
  "types": "./esm/types.d.ts",
  "exports": {
    "import": {
      "types": "./esm/types.d.ts",
      "default": "./esm/index.js"
    },
    "require": {
      "types": "./dist/types.d.ts",
      "default": "./dist/index.js"
    }
  },
```
Or those files are built when running `yarn add` / `npm install`?

Reply:
`esm` makes sense but the problem with `dist` is we need it in Github so we can require the files with JS deliver. For example:

```html
<script src="https://cdn.jsdelivr.net/gh/manuelmhtr/countries-and-timezones@latest/dist/index.min.js" type="text/javascript"></script>
```

Reply:
That's still possible. We'd just have to switch to the NPM package URL scheme instead:

```html
  <script src="https://cdn.jsdelivr.net/npm/countries-and-timezones@latest/dist/index.min.js" type="text/javascript"></script>
```

**Examples**

* [GitHub source demo](https://codepen.io/jcfranco/pen/qBeZdBM)
* [NPM package demo](https://codepen.io/jcfranco/pen/LYwNVYg)

Reply:
Per our conversation, I'll keep `dist` and we can plan to drop for v4.0 to avoid a breaking change (as the `gh` URL is used in doc examples). I'll go ahead and update my PRs accordingly.